modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -9034,6 +9103,22 @@ public void testPostRelatedObjectEntryInDifferentCompany()
 		);
 	}
 
+	@Test
+	public void testPostScopeScopeKeyPageWithSiteExternalReferenceCode()
+		throws Exception {
+
+		Group group = _addObjectEntryAndGetGroup();
+
+		JSONObject postJSONObject = _invoke(
+			Http.Method.POST, group.getExternalReferenceCode(),
+			JSONUtil.put(
+				_OBJECT_FIELD_NAME_1, _OBJECT_FIELD_VALUE_1
+			).toString());
+
+		Assert.assertEquals(
+			_OBJECT_FIELD_VALUE_1, postJSONObject.getInt(_OBJECT_FIELD_NAME_1));
```

Comment:
Is there a way to inline/merge the invoke+assert?

Can you look at how `testPostCustomObjectEntryWithDateTimeObjectField` was implemented and check if it is possible?

Reply:
Can you analyze if you can do the same for the `put` method?